queries/highlights.scm

```diff
@@ -23,13 +23,21 @@
   name: (identifier) @type)
 (enum_declaration
   name: (identifier) @type)
+(type_parameter
+  (identifier) @type)
```

Comment:
I'm ok with this for now, but I think it'd be even better to change the grammar to alias `identifier` as a `type_identifier` inside of the `type_parameter` rule; then we wouldn't need this. If you're interested, we could do that as part of this PR, but we could also merge as-is.

Reply:
@maxbrunsfeld, Thanks for the suggestion. I have alias `identifier` to `type_indentifier`. Please take a look.